portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase

```diff
@@ -86,6 +86,11 @@ definition {
 			Segmentation.viewSegment(
```

Comment:
Hey @rafaellajordao ! I think it's missing to remove the Segment.viewSegment macro from the tests.

Reply:
yeah! i will fix :+1: